AirCasting/SDSync/ViewsFlow/ClearingSDCard/ClearingSDCardViewModel.swift

```diff
@@ -1,18 +1,95 @@
 // Created by Lunar on 06/12/2021.
 //
 import Foundation
+import CoreBluetooth
+
+enum ClearingSDCardError {
+    case undefined
+    case noConnection
+    case noClearing
+}
 
 protocol ClearingSDCardViewModel: ObservableObject {
+    var isClearingCompleted: Published<Bool>.Publisher { get }
+    var shouldDismiss: Published<Bool>.Publisher { get }
     var presentNextScreen: Bool { get set }
     var isSDClearProcess: Bool { get set }
+    func clearSDCard()
+    func getAlertTitle() -> String
+    func getAlertMessage() -> String
```

Comment:
using our new system we can do `var alert: AlertInfo { get }` (if it's merged in, not sure).

If not, maybe we can omit `get` here and make it as `var`? We can use computed vars in implementation part